### DISCUSSION TITLE
Evapotranspiration in NOAH-MP

### DISCUSSION THREAD

**Discussion by Ghp1995** [NCAR/hrldas#225] [Q&A] [answered]
Hello,

Could you please let me know which variable in the restart file of NOAH_MP represent evapotranspiration?

**Comment by cenlinhe** [comment, marked as answer]
These is no direct variables related to ET in restart file. The variables are diagnostic fluxes, so they are only in the output history file: ECAN + EDIR + ETRAN = total ET
ECAN: canopy water evaporation
EDIR: soil evaporation
ETRAN: transpiration

**Reply by Ghp1995** [reply]
Thank you so much, this is very helpful.

**Reply by cenlinhe** [reply]
Just a further clarification: both variables (ECAN and EDIR) are the net water fluxes (i.e., evaporation+sublimation - dew - frost).

**Reply by Ghp1995** [reply]
Hello @cenlinhe,
I have a question regarding the units of variables.

If my time steps are like this:
 FORCING_TIMESTEP = 3600
 NOAH_TIMESTEP    = 1800
 OUTPUT_TIMESTEP  = 86400
 RESTART_FREQUENCY_HOURS = 24

What will be the units of QIN, QDIS, UDRUNOFF, RUNSUB and RUNSRF in restart file? I have added some of this variables to restart file following the provided tutorial at https://github.com/NCAR/hrldas/blob/master/tutorial/Note3_Output_Additional_Variables.ipynb

**Reply by cenlinhe** [reply]
1. I do not think QIN, QDIS, RUNSUB, and RUNSRF are the correct variable names used in restart file. They are variables used inside the code not the output variable names inside the restart file.
2. For UDRUNOFF and SFCRUNOFF, the unit is mm (accumulated total since the model start time point).
